scripts/release.py

```diff
@@ -522,6 +522,20 @@ def generate_email(start_ref, end_ref, release_date=None):
     return textwrap.dedent(email).strip()
 
 
+def cut_release_branch(release_date=None):
+    if release_date is None:
+        release_date = default_release_date()
+
+    release_branch_name = "rc/{date}".format(date=release_date)
+
+    git.checkout('master')
```

Comment:
I'm guessing we need some try/excepts around these statements? For example, if there are unstaged changes detected..